src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -113,10 +113,17 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
     setShowDeleteEventModal();
   };
 
-  const dropdownVenueList = venue?.rooms?.map(({ title, template }) => ({
-    name: title,
-    template: template,
-  }));
+  const dropdownVenueList =
+    venue?.rooms?.reduce(
+      (obj, room) => ({
+        [room.title]: {
+          name: room.title,
+          template: template,
+        },
+        ...obj,
+      }),
+      {}
+    ) ?? ALWAYS_EMPTY_OBJECT;
 
   const parentRoom = venue?.rooms?.find(({ title }) => title === event?.room);
```

Comment:
`Object.fromEntries()` here?  `useMemo()` for both variables?

Reply:
done eef5fce